labchecker/plugins/recover.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+
+
+class Recover(Enum):
+    SECURITY_GROUP = 1
+    SECURITY_GROUP_19885 = 2
+    SECURITY_GROUP_22 = 3
+    SECURITY_GROUP_ICMP = 4
+    NETWORK = 5
+    NETWORK_SUBNET = 6
+    NETWORK_SUBNET_CIDR = 7
+
+
+RECOVER_MAPS = {
```

Comment:
@wangxiyuan it has been changed to capital letter.

Reply:
++，BTW，Instead collect all recover functions here, I think move  each one to related plugin is better. It could make the code  more readable and decouple.

Any way, the pr looks good enough.